docs/website/docs/reference/optimization-options.md

```diff
@@ -0,0 +1,54 @@
+# Optimization Options
+
+This page documents various supported flags for optimizing IREE programs. Each
+is presented with its English name, flag to enable/disable and default state.
+
+These flags can be passed to the:
+
+* `ireec` command line tool
+* `extra_args=["--flag"]` argument to `iree.compiler.tools` Python wrappers
+* In-process Python compiler API
+  `iree.compiler.transforms.ireec.CompilerOptions("--flag", "--flag2")`
+  constructor
+* `ireeCompilerOptionsSetFlags()` compiler C API function
+
+## High level program optimizations
+
+### Constant Evaluation (`--iree-opt-const-eval` (off))
```

Comment:
Can you swap these headings to `Sentence case` (not `Title Case`) for consistency with the rest of the website? https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings